sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/TestUtils.java

```diff
@@ -133,6 +133,17 @@ public static ServiceBusMessage getServiceBusMessage(String body, String message
         return message;
     }
 
+    public static ServiceBusMessage getServiceBusMessageBinary(String messageTrackingValue, int position, int messageSize) {
```

Comment:
Why? Our strings are encoded as byte arrays, so I'm not sure what extra value this brings.